README.md

```diff
@@ -0,0 +1,166 @@
+        <img src="./docs/images/reddit.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Youtube" href="https://www.youtube.com/user/blockmaniaTV">
+        <img src="./docs/images/youtube.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Patreon" href="https://www.patreon.com/Terasology">
+        <img src="./docs/images/patreon.jpg" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Terasology Forum" href="https://forum.terasology.org">
+        <img src="./docs/images/forum.png" width="48px"/>
+    </a>
+</p>
+
+
+## Installation
+
+<table>
+    <tr>
+        <td></td>
+        <th>Minimum Requirements</th>
+    </tr>
+    <tr>
+        <td>System (OS)</td>
+        <td>Windows, MacOS, Linux (64 bit)</td>
+    </tr>
+    <tr>
+        <td>Processor (CPU)</td>
+        <td>dual-core CPU</td>
+    </tr>
+    <tr>
+        <td>Memory (RAM)</td>
+        <td>4 GB</td>
+    </tr>
+    <tr>
+        <td>Graphics* (GPU)</td>
+        <td style="vertical-align:top">
+            Intel HD Graphics (Gen 7)<br/>
+            GeForce 8xxx series (or higher) or<br/>
+            Radeon HD 2000 series (or higher)<br/>
+            with OpenGL 3.3
+        </td>
+    </tr>
+    <tr>
+        <td>Storage (HDD)</td>
+        <td>1 GB</td>
+    </tr>
+</table>
+
+\* _Please note, that if you have both integrated (chip) and dedicated (card) graphics, you should make sure that you're actually using your dedicated graphics when running Terasology._
+
+Internet connectivity is required for downloading Terasology via the Launcher, afterwards playing offline is possible.
+
+For easy game setup (recommended) you can use our launcher - [download it here](https://terasology.org/download.html).
+
+For more information about playing, like hot keys or server hosting, see the [dedicated page](docs/Playing.md) or check out the [modules](docs/Modules.md).
+
+
+### Alternative Installation Methods
+
+If you already have a Java Development Kit (JDK) installed, you may use a direct download release as an alternative to using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java version 11 is required.
+
+Direct download stable builds are uploaded to [our release section here on GitHub](https://github.com/MovingBlocks/Terasology/releases) while the cutting-edge develop version can be downloaded direct [here from our Jenkins](https://jenkins.terasology.io/teraorg/job/Terasology/job/Omega/job/master/lastSuccessfulBuild/artifact/distros/omega/build/distributions/TerasologyOmega.zip).
+
+
+## Development
+
+Development is possible on all common platforms (Windows, Linux, MacOS) as long as the JDK is properly set up.
+
+### Requirements
+
+Technical Requirements:
+- Java SE Development Kit (JDK) 11. It may be possible to use a later version of the JDK for local development, but the CI will verify against this baseline version.
```

Comment:
I would either omit the latter part of "It may be possible to use a later version of the JDK [...]" as it is kind of an _advanced topic_ you won't need for general setup, and it is definitely not recommended to start that way, or link to https://github.com/MovingBlocks/Terasology/issues/3976 for further details. 

We should probably keep "[T]he CI will verify against this baseline version."

Reply:
okay the way I changed it?